### PR TITLE
Update the catalyst 101 doc

### PR DESCRIPTION
**Pull request by aesharpe**
### Describe the Change

The handbook hasn't been updated in a little while, so I: 
- added some new links
- updated references to the work teams and internal teams
- changed the name business teams to internal teams 
- added info about sprint planning and check-in meetings
- remove sprint review meetings
- added info about continuous feedback
- a few other little tweaks

There's probably more I could have updated or changed, but I tried to keep this relatively contained.

### CODE REVIEW

File: docs/catalyst_101.md [hunk 1]
```diff
@@ -73,14 +73,14 @@ After taxes, contractors and members receive the same hourly rate. Employee-memb
...
+| **Meetings**         | - [M/W/Th stand-ups](mwth-stand-up-meetings-15-20-min)<br>- [Monday action-items](weekly-check-in--action-items-10-30-min)<br>- [Quarterly advisory committee meetings](quarterly-advisory-committee-meeting-1-hours)<br>- [Internal team meetings](internal-team-meetings)<br>- [Work team sprint meetings](work-team-meetings)<br>- [3 and 6 month feedback](contractor-feedback-meeting)<br>- [Financial check-in](monthly-financial-check-ins) (after 3 months) | - All contractor meetings, plus:<br>- [Annual or Special Member Meeting](member-meetings)<br>- [Board Meeting](weekly-board-meetings) (until 10+ members)<br>- [Certain feedback meetings for contractors](contractor-feedback-meeting)<br>- [Certain Feedback meetings for members](annual-member-feedback)                                                                                                                                                                                         |
```
Comment (aesharpe) [inline docs/catalyst_101.md:79]: This looks like a giant change, but the only think I added here is a row for the work team sprints: 

```
[Work team sprint meetings](work-team-meetings)<br>- 
```

File: docs/catalyst_101.md [hunk 2]
```diff
@@ -173,76 +176,69 @@ We don't prescribe how each team should organize its work. However, we expect ea
...
-- **Hiring & New Employee Management**
+#### Internal Teams
+
+- **Admin**: Run taxes, invoicing, payroll.
+- **Internal Management**: Curate PM tools and processes, track internal capacity, HR, documentation of processes.
+- **Community Development**: Build community around PUDL development and data use. Improve user experience, communicate our mission to the world, and interact with users and contributors.
+- **Grant Development**: Identify grant opportunities that are relevant to PUDL and
+  Catalyst.
+- **Business Development**: Identify opportunities and develop new relationships that lead to revenue-generating, mission-aligned client projects. Maintain relationships with existing clients.
+- **Hiring & New Employee Management**: Ad hoc team for hiring and onboarding new
```
Comment (jdangerx) [inline docs/catalyst_101.md:240]: We're missing PUDL Pat here, should we include it?

File: docs/catalyst_101.md [hunk 3]
```diff
@@ -280,37 +276,40 @@ All meetings have pre-designated facilitators and note-takers that are specified
...
-(quarterly-advisory-committee-meeting-1-hours)=
+#### Internal Team Meetings - Team Specific (time and occurrence differs)
 
-#### Quarterly Advisory Committee Meeting (1+ hours)
+**Attendees**: Team members.
+**Notes:** See notes documents for specific teams: [Internal teams](https://drive.google.com/open?id=1KuF1sJGQMWmODpD5LhnWVN61Ymbmg9xR)
 
-**Attendees**: Contractors & Members
-**Notes:** [Advisory Committee](https://drive.google.com/open?id=1f7y2csPj4rtJJoYJW05kwHyY7-sD_5Fb)
+- Catalyst organizes internal tasks into a series of teams that meet on a semi-regular basis to discuss and execute issues related to their domain. See [above](internal-teams) for a breif description, or refer to the [Teams Index](https://docs.google.com/document/u/0/d/1B9wzU3IuNlJbqG-4XpcpJnIpXhq8IRzf6m4lJnxXVvo/edit) for a list of current teams, their responsibilities, and how often they meet.
```
Comment (jdangerx) [inline docs/catalyst_101.md:303]:
```suggestion
- Catalyst organizes internal tasks into a series of teams that meet on a semi-regular basis to discuss and execute issues related to their domain. See [above](internal-teams) for a brief description, or refer to the [Teams Index](https://docs.google.com/document/u/0/d/1B9wzU3IuNlJbqG-4XpcpJnIpXhq8IRzf6m4lJnxXVvo/edit) for a list of current teams, their responsibilities, and how often they meet.
```

File: docs/catalyst_101.md [hunk 4]
```diff
@@ -280,37 +276,40 @@ All meetings have pre-designated facilitators and note-takers that are specified
...
+- Teams are not mandatory, but contractors are encouraged and members are expected to participate in at least one.
```
Comment (jdangerx) [inline docs/catalyst_101.md:304]: Would we approve a member candidate who refused to be in any internal teams? Would we be happy with a member who refused to be in any internal teams? From my perspective it seems like we wouldn't... but maybe that's a bigger discussion.

File: docs/catalyst_101.md [hunk 5]
```diff
@@ -280,37 +276,40 @@ All meetings have pre-designated facilitators and note-takers that are specified
...
 
-- Every Quarter Catalyst holds an Advisory Committee (AC) meeting during which we seek direction and feedback from the [members of our AC](https://catalyst.coop/advisory-board/) . The AC is composed of individuals that span the tech and environmental sectors. We generally seek information about funding opportunities and share work updates.
+(quarterly-advisory-committee-meeting-1-hours)=
```
Comment (jdangerx) [inline docs/catalyst_101.md:306]:
```suggestion
```

No more advisory meeting, right? We can remove this label?

File: docs/catalyst_101.md [hunk 6]
```diff
@@ -319,49 +318,50 @@ All meetings have pre-designated facilitators and note-takers that are specified
...
 - Until we have more than 10 members, the Board of Directors is made up of all of the members. Board meetings are the venue where decisions about the strategic direction and regular operation of the cooperative are made. Any member of the board can go in and add agenda items.
-
-#### Weekly Refi Call (1 hour)
-
-**Attendees:** Appointed person(s) or volunteer(s)
-**Notes:** n/a
-
-- Catalyst participates in weekly Fossil Fuel Refinance Project calls. These calls are a consortium of energy policy folks working on refinancing strategies to close coal plants and other fossil fuel-driven generation assets. Usually one-two Catalyst members join these calls.
-- For the call, you’ll need to solicit updates from the Catalyst team in order to communicate our weekly progress to the refinance team; coordinating collaboration with the refinance team; and taking notes on relevant updates from the call to convey to the Catalyst team.
+- Board meetings may be cancled if there is nothing on the agenda.
```
Comment (jdangerx) [inline docs/catalyst_101.md:325]:
```suggestion
- Board meetings may be canceled if there is nothing on the agenda.
```

File: docs/catalyst_101.md [hunk 7]
```diff
@@ -319,49 +318,50 @@ All meetings have pre-designated facilitators and note-takers that are specified
...
 
+(monthly-member-check-ins)=
+
+#### Monthly Member Check-ins (30 min)
+
+**Attendees:** Member and check-in facilitator.
+**Notes:** Notes are private and only shared with the member and meeting facilitator.
+
+- Catalyst members participate in monthly check-in calls where they
+  touch base on their annual goals and general feelings. These calls are intended to provide members with an outlet for expressing excitement or concern about anything going on.
```
Comment (jdangerx) [inline docs/catalyst_101.md:355]:
```suggestion
  touch base on their annual goals and general feelings. These calls are intended to help members keep track of their goals and make sure they are supported in achieving them.
```

Though you are right in that a lot of the support turns out to be providing an outlet to talk about work feels.

**Review by jdangerx — CHANGES_REQUESTED** [review]
Few small changes but thanks for updating!

File: docs/catalyst_101.md (hunk 2 above)
Comment (aesharpe) [reply docs/catalyst_101.md:240]: I would say no. We can add that later if it becomes more of a thing.

File: docs/catalyst_101.md (hunk 4 above)
Comment (aesharpe) [reply docs/catalyst_101.md:304]: I think we would definitely let people opt out of teams if we got large enough. Even now, if someone just wanted to code I think that would be fine--we have been talking about trying to reduce team size anyways.

**Review by jdangerx — APPROVED** [review]

**Pull request merged**